tools/ci_build/github/apple/coreml_supported_mlprogram_ops.md

```diff
@@ -20,3 +20,4 @@ Keep in sync with doco generated from /docs/execution-providers/CoreML-Execution
 |ai.onnx:Sub||
 |ai.onnx:Sigmoid||
 |ai:onnx:Tanh||
+|ai:onnx:Concat||
```

Comment:
Let's keep these alphabetically sorted so it's easier for the reader to find a specific op.